### PR TITLE
Ed/update identity styling

### PR DESCRIPTION
**Pull request by mosster**

### CODE REVIEW

File: src/app/account-select/account-select.component.html [hunk 1]
```diff
@@ -1,42 +1,37 @@
-<div class="d-flex flex-column" *ngIf="hasUsers">
-  <ul class="list-group mt-7px mb-30px saved-seeds-list">
-    <span class="saved-seeds-header d-flex align-items-center"
-      ><span>{{componentTitle}}</span></span
+<div class="section--accounts" *ngIf="hasUsers">
+  <div class="section--accounts__header margin-bottom--medium">
```
Comment (lazynina) [inline src/app/account-select/account-select.component.html:2]: We probably want a little spacing between component title and the user selection.

File: src/app/account-select/account-select.component.html [hunk 2]
```diff
@@ -1,42 +1,37 @@
...
-            class="mr-10px"
+            class="avatar--rounded avatar--large margin-right--xsmall"
+            src="assets/placeholder-account-image.png"
           />
+          {{ item.key | truncateAddressOrUsername }}
         </div>
-      </li>
-    </div>
+        <div *ngIf="item.value.username" class="d-flex align-items-center">
+          <div [appAvatar]="item.key"></div>
```
[the remark this answers is not in the document]
Comment (lazynina) [reply src/app/account-select/account-select.component.html:21]: Additionally, I'm not crazy about the white on the blue background, but I'll leave that to you to spruce up as you see fit.

File: src/app/log-in-options/log-in-options.component.html [hunk 3]
```diff
@@ -1,46 +1,46 @@
...
+      class="button--primary--outline button--medium margin-bottom--medium"
+      [routerLink]="['/sign-up']"
+      queryParamsHandling="merge"
+    >
+      Sign up with DeSo seed
+    </button>
+    <a
+      data-control-name="log-in-cta"
+      data-control-login-option="log-in-with-deso-seed"
+      class="text--small" [routerLink]="['/load-seed']" queryParamsHandling="merge"
```
Comment (lazynina) [inline src/app/log-in-options/log-in-options.component.html:41]: The load seed page has no padding on the sides. Let's address this in a separate PR.

File: src/styles.scss [hunk 4]
```diff
@@ -13,90 +17,475 @@ $blue: #007bff;
...
+// FONTS = Families
+//-------------------------------
+$font-primary: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+
 @font-face {
-  font-family: 'Eina01';
-  src: url('./assets/fonts/Eina01-Bold.ttf') format('truetype');
-  font-weight: bold;
+  font-family: 'color-emoji';
+  src: local('Apple Color Emoji'), local('Segoe UI Emoji'),
```
Comment (lazynina) [inline src/styles.scss:47]: we like to package up the fonts in our assets folder so we control everything for identity. fewer external touchpoints. Are you able to package any of these up as a ttf? I'm not a font expert and maybe this local stuff is ok?

File: src/styles.scss [hunk 5]
```diff
@@ -261,463 +650,6 @@ $error: #ff6939;
...
-.mx-40px {
-  margin-left: 40px;
-  margin-right: 40px;
-}
-.my-40px {
-  margin-top: 40px;
-  margin-bottom: 40px;
-}
-
-.m-0auto {
```
Comment (lazynina) [inline src/styles.scss:718]: removing all of these is good, but we just gotta make sure we get rid of all uses as well.

**Review by lazynina — CHANGES_REQUESTED** [review]

File: src/app/banner/banner.component.html [hunk 6]
```diff
@@ -1,22 +1,13 @@
...
-        <img
-          class="ml-15px position-relative"
-          src="/assets/lock-green.svg"
-          style="top: -2px"
-        />
-        <span class="fc-green ml-5px">{{
-          globalVars.environment.hostname
-        }}</span>
-      </div>
+<div class="section--header" *ngIf="!globalVars.webview">
```
Comment (lazynina) [inline src/app/banner/banner.component.html:1]: Banner is looking a bit bunched up on mobile: 
image

Comment (lazynina) [reply src/app/banner/banner.component.html:1]: Actually everything is a bit wonky on mobile:
image

File: src/styles.scss (hunk 5 above)
Comment (mosster) [reply src/styles.scss:718]: Yup, will do a cleanup past towards the end once I've gone through most of the screens & states.

File: src/styles.scss (hunk 4 above)
Comment (mosster) [reply src/styles.scss:47]: I actually deleted the Eina font, and this Emoji thing was already here. I'll clean this up at the end if it's not being used anywhere (I'll likely remove emojis anyways if I do spot any).

File: src/app/log-in-options/log-in-options.component.html (hunk 3 above)
Comment (mosster) [reply src/app/log-in-options/log-in-options.component.html:41]: Yup, will do a mobile pass at a later point to get everything responsive with padding on the sides.

File: src/app/account-select/account-select.component.html [hunk 7]
```diff
@@ -1,42 +1,37 @@
...
+        </div>        
+      </div>
+      <img
+        *ngIf="!hideLoginMethod"
+        [src]="
+          getLoginIcon(
+            accountService.getLoginMethodWithPublicKeyBase58Check(item.key)
+          )
+        "
+        width="22px"
```
[the remark this answers is not in the document]
Comment (mosster) [reply src/app/account-select/account-select.component.html:32]: Should be fixed!

File: src/app/account-select/account-select.component.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (mosster) [reply src/app/account-select/account-select.component.html:21]: Yeah removed the white background. Going to tweak this section a little bit more.

I'll do a pass on images and avatars as well...

File: src/app/banner/banner.component.html (hunk 6 above)
Comment (mosster) [reply src/app/banner/banner.component.html:1]: Yup, will do a mobile pass.

File: src/app/log-in/log-in.component.html [hunk 8]
```diff
@@ -1,74 +1,94 @@
...
-        Buy, sell, and send coins on my behalf
-        <div
-          class="fs-14px text-muted"
-          *ngIf="globalVars.accessLevelRequest < 4"
-        >
-          <b>{{ globalVars.hostname }}</b> will require approval to buy, sell,
-          or send coins
+      <li class="section--access__item display--flex">
+        <div>
+          <span *ngIf="globalVars.accessLevelRequest >= 3"><img src="assets/image-check.svg" width="30px"/></span>
```
Comment (lazynina) [inline src/app/log-in/log-in.component.html:60]:
```suggestion
          <span *ngIf="globalVars.accessLevelRequest === 4"><img src="assets/image-check.svg" width="30px"/></span>
```

**Review by lazynina — APPROVED** [review]

**Pull request merged**